packages/0x.js/src/types.ts

```diff
@@ -384,29 +385,4 @@ export interface OrderStateInvalid {
 
 export type OrderState = OrderStateValid|OrderStateInvalid;
 
-export type OnOrderStateChangeCallback = (orderState: OrderState) => void;
-
-export interface TxData {
-    from?: string;
-    gas?: number;
-    gasPrice?: BigNumber;
-    nonce?: number;
-}
-
-export interface TxDataPayable extends TxData {
-    value?: BigNumber;
-}
-
-export interface TransactionReceipt {
-    blockHash: string;
-    blockNumber: number;
-    transactionHash: string;
-    transactionIndex: number;
-    from: string;
-    to: string;
-    status: null|0|1;
-    cumulativeGasUsed: number;
-    gasUsed: number;
-    contractAddress: string|null;
-    logs: Web3.LogEntry[];
-} // tslint:disable:max-file-line-count
+export type OnOrderStateChangeCallback = (orderState: OrderState) => void; // tslint:disable:max-file-line-count
```

Comment:
Can we put `// tslint:disable:max-file-line-count` on it's own line at the very end?